plugins/core.go

```diff
@@ -1,20 +1,21 @@
 package plugins
 
 import (
+	"github.com/iotaledger/goshimmer/plugins/metrics"
```

Comment:
rename the `metrics` to `metricsCollector`, which is the name of this plugin, or the other way around.

Reply:
`metrics` it is